README.rst

```diff
@@ -7,7 +7,7 @@
 ==============================================================================
 pypiserver - minimal PyPI server for use with pip/easy_install
 ==============================================================================
-|pypi-ver| |travis-status| |dependencies| |downloads-count| |python-ver| \
+|pypi-ver| |travis-status| |dependencies| |python-ver| \
 |proj-license|
```

Comment:
Might as well pull `|proj-license|` up onto the same line so we don't need the backslash line continuation

Reply:
Done!